gcc/rust/backend/rust-compile-resolve-path.cc

```diff
@@ -106,42 +113,17 @@ ResolvePathRef::attempt_constructor_expression_lookup (
 }
 
 tree
-ResolvePathRef::resolve (const HIR::PathIdentSegment &final_segment,
-			 const Analysis::NodeMapping &mappings,
-			 location_t expr_locus, bool is_qualified_path)
+ResolvePathRef::resolve_with_node_id (
+  const HIR::PathIdentSegment &final_segment,
+  const Analysis::NodeMapping &mappings, location_t expr_locus,
+  bool is_qualified_path, NodeId resolved_node_id)
```

Comment:
I have a feeling the is_qualified_path is no longer used here now which is neat

Reply:
ah I kept it cause we need it for `query_compile`